app/backend/src/couchers/models.py

```diff
@@ -252,6 +249,11 @@ def display_last_active(self):
         """
         return self.last_active.replace(minute=(self.last_active.minute // 15) * 15, second=0, microsecond=0)
 
+    def phone_is_verified(self):
+        return self.phone_verification_verified is not None and now() - self.phone_verification_verified < timedelta(
+            days=356
+        )
```

Comment:
Could you move the `timedelta` into `constants.py`, and I think make it 2 years for now.